tmt-app/src/Components/Quiz.jsx

```diff
@@ -17,4 +29,56 @@ function Quiz() {
   );
 }
 
+function Text({ step }) {
+  const textContent = {
```

Comment:
looks like this should be an array

Reply:
yup, either we use indexes to access the an array of data, or turn this into a proper hash map where we can use a predefined set of available stages as strings we can use in the url rather than numbers